components/sup/src/event/stream.rs

```diff
@@ -11,7 +11,7 @@ use tokio::{prelude::Stream,
             runtime::current_thread::Runtime};
 
 /// All messages are published under this subject.
-const HABITAT_SUBJECT: &str = "habitat";
+const HABITAT_SUBJECT: &str = "habitat.event.healthcheck";
```

Comment:
We send out more events than just healthchecks, though... if we're going to have different subjects per event, we'll need to handle that a little differently.

Reply:
Thanks for catching that! I will follow up with the automate team and see what they expect.